packages/manager/modules/pci/src/projects/project/serving/namespace/models/add/add.controller.js

```diff
@@ -0,0 +1,93 @@
+import get from 'lodash/get';
+
+export default class PciServingNamespaceModelsAddController {
+  /* @ngInject */
+  constructor(
+    $translate,
+    OvhManagerPciServingModelsService,
+    PciStoragesContainersService,
+  ) {
+    this.$translate = $translate;
+    this.OvhManagerPciServingModelsService = OvhManagerPciServingModelsService;
+    this.PciStoragesContainersService = PciStoragesContainersService;
+  }
+
+  $onInit() {
+    this.isAdding = false;
+
+    this.model = {
+      id: null,
+      storagePath: null,
+      flavor: null,
+      kind: null,
+      preset: null,
+    };
+
+    this.presets = [
+      {
+        id: '3b2c0bdf8b5349e8bafb79b76419b3f0',
+        name: 'Text Sentiment Analysis',
+        link: 'https://market-place.ai.ovh.net/#!/apis/3072602f-7adb-4398-b260-2f7adb2398f9/pages/ae10410c-a12c-495d-9041-0ca12ca95da6',
+      },
+      {
+        id: '3b2c0bdf8b5349e8bafb79b76419b3f2',
+        name: 'Text Sentiment Analysis 2',
+        link: 'https://market-place.ai.ovh.net/#!/apis/3072602f-7adb-4398-b260-2f7adb2398f9/pages/ae10410c-a12c-495d-9041-0ca12ca95da6',
+      },
+    ];
+
+    this.flavors = [
+      {
+        id: '3b2c0bdf8b5349e8bafb79b76419b3f0',
+        name: 'Preview',
+        type: 'CPU',
+        infos: {
+
+        },
+        price: 0,
+      },
+    ];
```

Comment:
Should be returned by API

Reply:
yep, fixed 9eb0962dad0527f7e808bd6b11ebe3c3cfec136e